src/DateInput/DateInput.tsx

```diff
@@ -52,7 +52,10 @@ const popperPlacementPositions = {
   TOP_START: "top-start",
 } as const;
 
-const dateType = PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.instanceOf(moment)]);
+const dateType = PropTypes.oneOfType([
+  PropTypes.instanceOf(Date),
+  PropTypes.instanceOf(moment as any),
```

Comment:
Curious why the `as any` is necessary here. Is there a whitelist with respect to what can be passed to `PropTypes.instanceOf`?

Reply:
Yeah it accepts types that can be instantiated with `new`